### ISSUE TITLE
Alert save method fails in v2 API endpoint

### ISSUE THREAD

**Issue by DrkDigglr** [carbonblack/cbapi-python#202]
When I try to update the status of an Alert object via the save method it uses the v2 API path, resulting in an "ObjectNotFoundError":
```
cbapi.errors.ObjectNotFoundError: Received 404 (Object Not Found) for https://xxxxx.my.cbcloud.de/api/v2/alert/3642f15f-e747-454c-bea2-f6bb6abfe5f8: <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>404 Not Found</title>
<h1>Not Found</h1>
<p>The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.</p>
```
Doing a GET request to that URL results in the same error. Swapping the "v2" out for "v1" and doing a GET request results in a 405 (method not allowed) indicating that the endpoint is there at the very least.

Issue [https://github.com/carbonblack/cbapi-python/issues/158](https://github.com/carbonblack/cbapi-python/issues/158) appears to be related. I will try reverting back to an older version of cbapi for now.

**Comment by DrkDigglr** [comment]
After reverting back to cbapi version 1.4.2 the save method works but I different error:
```
cbapi.errors.ServerError: Received error code 500 from API: Unhandled exception. Check logs for details.
```
While it has updated the status, I'm not entirely confident in how this works and would prefer to use the latest stable version of this library.

This is all being executed against an instance in Carbon Black Response Cloud.

Thanks in advance for any help.

**Comment by DrkDigglr** [comment]
I've noticed that you've release version 1.6.0 that is supposed to address the issues described above. It does not. Am I missing something here? Is there anything that needs to be done on CB Response Cloud?

**Comment by avanbrunt-cb** [comment]
@DrkDigglr we have published a fix with 1.6.1

**Comment by DrkDigglr** [comment]
Thank you!
